WcaOnRails/app/models/competition.rb

```diff
@@ -66,10 +65,10 @@ class Competition < ActiveRecord::Base
     self.persisted? || (name.length <= MAX_NAME_LENGTH && name =~ VALID_NAME_RE)
   end
   validates :name, length: { maximum: MAX_NAME_LENGTH },
-                   format: { with: VALID_NAME_RE, message: INVALID_NAME_MESSAGE }
+                   format: { with: VALID_NAME_RE, message: I18n.t('competitions.errors.invalid_name_message') }
```

Comment:
A block in this context could be a hash, so Ruby is not happy, but `proc { I18n.... }` does the trick :)
(not using a lambda because we don't necessarily know the number of arguments)